drivers/clock_control/clock_control_mcux_ccm_rev2.c

```diff
@@ -63,10 +63,16 @@ static int mcux_ccm_get_subsys_rate(const struct device *dev,
 	instance = (clock_name & IMX_CCM_INSTANCE_MASK);
 	switch (peripheral) {
 #ifdef CONFIG_I2C_MCUX_LPI2C
+#if defined(CONFIG_SOC_SERIES_IMXRT118X)
```

Comment:
Should we handle all clock cases here, or just LPI2C0102?

Reply:
Here is not only just LPI2C0102, all the lpi2c clock can be handle here. Only LPUART/LPSPI/LPI2C on RT1180 exist phenomenon two instances share the same clock root.
LPUART clock have been handle on basic environment, this PR handle lpi2c clock, then I will push lpspi support later, it will handle lpspi clock issue.